appinventor/codelabs/mole-mash/index.html

```diff
@@ -0,0 +1,242 @@
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Define a procedure to update the score">
+        <p>Our second prodecure will update the score of the game.</p>
+        <p>First, we need a variable to keep track of the score. Define a global variable named "score", and set the initial value to 0. This is what your block should look like:</p>
+        <img src="img/score_variable.png" style="max-width: 300px">
+        <p>Next, define a procedure named "UpdateScore". This procedure will change the text of the the label <b>ScoreLabel</b> to show the score of the game.</p>
+        <h2>Challenges</h2>
+        <ol>
+          <li>Try to code this procedure by yourself.</li>
+          <p>Hint: The completed UpdateScore procedure should look something like this:</p>
+          <img src="img/updatescore.png" style="max-width: 400px">
+        </ol>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Add a Timer">
+        <p>The next step is to keep the mole moving. Every time MoleTimer goes off, the procedure MoveMole is called, meaning that the Mole has moved to a new, random position.</p>
+        <p>From the <b>MoleTimer</b> drawer, drag out a <code>when MoleTimer.Timer</code> block, and from the <b>Procedures</b> drawer, drag a <code>call MoveMole</code> block into the <code>when MoleTimer.Timer</code> block.</p>
+        <img src="img/moletimer.png" style="max-width: 250px">
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Add a Mole Touch handler">
+        <p>The program should increment the score each time the mole is touched. Sprites, like canvases, respond to touch events.</p>
+        <p>Every time the Mole is touched, your program will increment the score, call the UpdateScore prodecure to show the new score, make the phone vibrate for 100 milliseconds, and call the MoveMole procedure so the mole moves right away.</p>
+        <h2>Challenges</h2>
+          <ol>
+            <li>Try to code this procedure by yourself.</li>
+            <p>Hint: The completed event handler should look something like this:</p>
+            <img src="img/mole_touch.png" style="max-width: 400px">
+            <li>Use typeblocking: typing to quickly create blocks.</li>
+            <p>To create a value block containing 100, just type 100 and press return</p>
+            <p>To create a MoveMole block, just type MoveMole and select the block you want from the list</p>
+          </ol>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Reset the score">
+        <p>The last thing we need to do is reset the score every time the Reset button is pressed.</p>
+        <p>From the <b>ResetButton</b> drawer, drag a <code>when ResetButton.Click</code> block into the workspace. Place a <code>set global score to</code> block inside, and set the global score to 0. Next, update the score my placing a <code>call UpdateScore</code> block after the <code>set global score to</code> block.</p>
```

Comment:
"my" should be "by"

update the score label
(We want to make sure they know the difference between updating the score itself and updating the label.)